static/js/components/dashboard/courses/GradeDetailPopup.js

```diff
@@ -3,57 +3,87 @@ import Dialog from 'material-ui/Dialog';
 import React from 'react';
 import R from 'ramda';
 import Icon from 'react-mdl/lib/Icon';
+import moment from 'moment';
 
 import type {
   Course,
   CourseRun,
+  ProctoredExamResult,
 } from '../../../flow/programTypes';
 import { formatGrade } from '../util';
 import {
   STATUS_NOT_PASSED,
   STATUS_PASSED,
+  DASHBOARD_FORMAT,
 } from '../../../constants';
+import type { GradeType } from '../../../containers/DashboardPage';
+import { EDX_GRADE } from '../../../containers/DashboardPage';
 
-const runGrade = (courseRun: CourseRun, isBestGrade: boolean): string|React$Element<*> => {
-  if (courseRun.final_grade) {
+const formatExamGrade = percentageGrade => percentageGrade * 100;
```

Comment:
What's the difference between this and `formatGrade`? If we want to omit the `%` we should still use `_.round` here